### PR TITLE
KurilenkoDZ1

### PR DESCRIPTION
**Pull request by Bellisa**

### CODE REVIEW

File: src/kurilenko.olesya/homework.spec.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import {
+  sumSalaries,
+  maxSalary,
+  multiplyNumeric,
+  countBy,
+  addClass,
+  addClass2
+} from './homework';
+
+var salaries = {
```
Comment (LimarenkoDenis) [inline src/kurilenko.olesya/homework.spec.ts]: pls use es6 syntax (const)

File: src/kurilenko.olesya/homework.spec.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  "Вася": 100,
+  "Петя": 300,
+  "Даша": 250
+};
+var menu = {
```
Comment (LimarenkoDenis) [inline src/kurilenko.olesya/homework.spec.ts]: pls declare vars in specific test scope

File: src/kurilenko.olesya/homework.spec.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+});
+
+describe('function multiplyNumeric ', () => {
+  test('should return { width: 400, height: 600, title: My menu }', () => {
+    expect(multiplyNumeric(menu)).toEqual(menuNew);
+  });
+});
+
+describe('function countBy', () => {
+  test('should return [1,2,3,4,5,6,7,8,9,10]', () => {
```
Comment (LimarenkoDenis) [inline src/kurilenko.olesya/homework.spec.ts]: pls extend tslint rules, 
`[2, 4, 6, 8, 10]`
`[1,2,3,4,5,6,7,8,9,10]`  take a look to spaces

File: src/kurilenko.olesya/homework.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    if (classes.indexOf(addName.trim()) === -1)
+      obj.className = obj.className + ' ' + addName.trim();
+    return obj;
+  }
+
+  export function addClass2(addName: string):(obj: { className: string }) => { className: string } {
+    
+    return function(obj): { className: string } {
+      return (obj.className.indexOf(addName.trim()) === -1)?
+        { className: obj.className+' '+addName.trim() }
```
Comment (LimarenkoDenis) [inline src/kurilenko.olesya/homework.ts]: use es6 syntax, template string

Comment (Bellisa) [reply src/kurilenko.olesya/homework.ts]: Do you mean `${obj.className} ${addName.trim()}` or 
i can use obj.className.concat(' ',addName.trim()) ?

File: src/kurilenko.olesya/homework.spec.ts (hunk 3 above)
Comment (Bellisa) [reply src/kurilenko.olesya/homework.spec.ts]: I didn't change the configuration file(tslint).

File: src/kurilenko.olesya/homework.spec.ts (hunk 1 above)
Comment (Bellisa) [reply src/kurilenko.olesya/homework.spec.ts]: sorry. I took objects from README. there var written/

**Pull request merged**